src/OpenColorIO/ColorSpaceSet.cpp

```diff
@@ -109,21 +119,53 @@ class ColorSpaceSet::Impl
 
     void add(const ConstColorSpaceRcPtr & cs)
     {
-        const std::string csName = StringUtils::Lower(cs->getName());
-        if (csName.empty())
+        const char * csName = cs->getName();

```

Comment:
Should this still be lowercase, for the below comparison?

Reply:
StringUtils::Compare is not case sensitive

Reply:
StringUtils::Compare is not case sensitive

Reply:
Ah, ok. Thanks for the clarification.